.github/workflows/ci.yml

```diff
@@ -359,204 +359,10 @@ jobs:
       - name: Check dirty git
         uses: ./.github/actions/check-dirty-git
 
-
-  fog-local-network-tests:
```

Comment:
Discussed in internal slack we think the main purpose of this test was to ensure that the fog_local_network.py script continues to function, the minting and burning test accomplishes this and does so faster.